mobile/src/libs/components/icon/libs/maps/icon-name-to-icon.map.ts

```diff
@@ -2,6 +2,8 @@ import BackArrowIcon from '#assets/img/icons/back-arrow.svg';
 import Backward from '#assets/img/icons/backward.svg';
 import BellIcon from '#assets/img/icons/bell.svg';
 import ChatIcon from '#assets/img/icons/chat.svg';
+import Close from '#assets/img/icons/close.svg';
+import Download from '#assets/img/icons/download.svg';
```

Comment:
```suggestion
import CloseIcon from '#assets/img/icons/close.svg';
import DownloadIcon from '#assets/img/icons/download.svg';
```

Reply:
Due to already existing icons, I think we need to add "Icon" everywhere in this file.

Reply:
I think it's better to remove 'Icon' everywhere and write them all in lowercase, @Liza-Veis , what do you think?

Reply:
Agree